### PR TITLE
Update instructions to cover new internal server data.neuro.polymtl.ca

### PR DESCRIPTION
**Pull request by kousu**

### CODE REVIEW

File: internal-server.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        * **Debian/Ubuntu**: `apt-get install git-annex`, but **you must be using Ubuntu 20.10** or **Debian Testing**
+        * if on an older system and can't upgrade, you can try [installing `conda`](https://docs.conda.io/en/latest/miniconda.html) and then `conda install git-annex`.
+    * **BSD**:
+        * OpenBSD: _untested_
+        * FreeBSD: _untested_
+        * NetBSD: _untested_
+    * **macOS**: `brew install git-annex`
+2. Make sure you have an ssh key.
+    * If not, run `ssh-keygen`. Your keys will be in the hidden folder `~/.ssh/`.
+4. Give your ssh public key -- that is, the contents of `~/.ssh/id_rsa.pub` or `~/.ssh/id_ed25519.pub`, making sure to use the **.pub** file -- to one of the server admins and ask for their consent to granting you access.
```
Comment (jcohenadad) [inline internal-server.md:38]:
```suggestion
3. Give your ssh public key -- that is, the contents of `~/.ssh/id_rsa.pub` or `~/.ssh/id_ed25519.pub`, making sure to use the **.pub** file -- to one of the server admins and ask for their consent to granting you access.
```

**Comment by jcohenadad** [comment]
@kousu i've tried your latest instructions https://github.com/neuropoly/data-management/pull/17/commits/f3fff71c4ff7f4483563ff14a089441098b31aa4, do you see my push?

<details><summary>git-annex sync</summary>

```
julien-macbook:~/temp/20210112_gitannex/data-single-subject $ git-annex sync
commit 
On branch jca/test
nothing to commit, working tree clean
ok
pull origin 
remote: Enumerating objects: 804, done.
remote: Counting objects: 100% (804/804), done.
remote: Compressing objects: 100% (287/287), done.
remote: Total 429 (delta 285), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (429/429), 28.44 KiB | 4.06 MiB/s, done.
Resolving deltas: 100% (285/285), completed with 244 local objects.
From data.neuro.polymtl.ca:datasets/data-single-subject
   7f37ec7..e1a4ac7  git-annex  -> origin/git-annex
ok
(merging origin/git-annex into git-annex...)
(recording state in git...)
push origin 
Enumerating objects: 31, done.
Counting objects: 100% (31/31), done.
Delta compression using up to 16 threads
Compressing objects: 100% (16/16), done.
Writing objects: 100% (20/20), 1.77 KiB | 1.77 MiB/s, done.
Total 20 (delta 11), reused 1 (delta 0)
To data.neuro.polymtl.ca:datasets/data-single-subject
   7f37ec7..d62ada9  git-annex -> synced/git-annex
 * [new branch]      jca/test -> synced/jca/test
ok

```

</details>

**Comment by kousu** [comment]
Current HEAD on the server:

```
git@data:~/repositories/datasets/data-single-subject.git$ git log -p
commit 7908de6ef8a6914cb193c28b0cbe51979ef5c825 (HEAD -> master, synced/master)
Author: Julien Cohen-Adad <jcohen@polymtl.ca>
Date:   Sat Sep 12 10:44:02 2020 -0400

    Matched excluded subjects for all DTI metrics

diff --git a/exclude.yml b/exclude.yml
index 92a05ff..9b4e8a2 100644
--- a/exclude.yml
+++ b/exclude.yml
@@ -5,11 +5,21 @@ csa_t1:
 csa_gm:
 - sub-juntendoAchieva  # repositionning (https://github.com/spine-generic/data-single-subject/issues/8)
 
+mtr:
+- juntendo750w  # TR of 62ms instead of 35ms
+- perform  # TR of 62ms instead of 35ms
+
 dti_fa:
 - juntendoAchieva  # no cardiac gating
 - perform  # strong aliasing artifact (https://github.com/spine-generic/data-single-subject/issues/10)
 - tokyo750w  # no cardiac gating
 
-mtr:
-- juntendo750w  # TR of 62ms instead of 35ms
-- perform  # TR of 62ms instead of 35ms
+dti_md:
+- juntendoAchieva  # no cardiac gating
+- perform  # strong aliasing artifact (https://github.com/spine-generic/data-single-subject/issues/10)
+- tokyo750w  # no cardiac gating
+
+dti_rd:
+- juntendoAchieva  # no cardiac gating
+- perform  # strong aliasing artifact (https://github.com/spine-generic/data-single-subject/issues/10)
+- tokyo750w  # no cardiac gating

commit 559473b9821ce6c881d34346cfba4c13434d5db0
Merge: 8a059ab 4cd64e3
Author: Julien Cohen-Adad <jcohen@polymtl.ca>
Date:   Fri Sep 11 21:51:23 2020 -0400

    Merge remote-tracking branch 'refs/remotes/origin/master'

commit 8a059ab6091e539d31862005f9383173625b416a
Author: Julien Cohen-Adad <jcohen@polymtl.ca>
Date:   Fri Sep 11 21:51:05 2020 -0400
```

**Comment by jcohenadad** [comment]
hum, so it means that you do not see my branch `jca/test`?

**Review by jcohenadad — APPROVED** [review]
this first iteration is already super helpful, thank you @kousu !

**Comment by kousu** [comment]
> hum, so it means that you do not see my branch `jca/test`?

Ah no that's there too. I missed that you wanted to look at that:

``
git@data:~/repositories/datasets/data-single-subject.git$  git branch
  git-annex
  jca/test
* master
  synced/git-annex
  synced/jca/test
  synced/master
```

**Pull request merged**